penpot/penpot
# :tada: :recycle: Refactor & Access Tokens (part 1)

This PR continues the on-going refactor of moving QUERIES and MUTATIONS to COMMANDS.
Additionally, there a little refactor related to sessions and authentication for make a bit of space for access-tokens.

**With this PR mutations and queries are now not used and will be deleted completely on 1.19**

How to test:

- Access tokens can be tested but they ara still not ready. You can test it creating the access token using a helper exposed on the namespace, but the best approach here I guess is wait until we have a proper UI for it.
- The rest of the refactor can be tested with basic flow, just perform all user actions (upload photo, upload team photo). I mean, if the basic flow of dashboard works, then everything should work as expected. **There are many unit tests for it so it has some automatic test coverage**.
- The user feedback can be tested just performing the feedback action
